react/components/EditorContainer/Sidebar/BlockEditor/BlockConfigurationList/Card/index.tsx

```diff
@@ -173,38 +177,42 @@ const Card = ({
           })}
       </div>
 
-      <ConditionTags conditions={conditions} />
+      {conditions.length > 0 && <ConditionTags conditions={conditions} />}
 
-      <div className="mt5">
-        <FormattedMessage
-          id={`admin/pages.editor.configuration.scope.${scope}.saved`}
-        >
-          {message => (
-            <div className="flex items-center">
-              {iconByScope[scope]}
-
-              <span className="ml2 f6">
-                {message}{' '}
-                <FormattedMessage
-                  id={`admin/pages.editor.configuration.scope.${scope}.context`}
-                >
-                  {message => <span className="fw5">{message}</span>}
-                </FormattedMessage>
-              </span>
-            </div>
-          )}
-        </FormattedMessage>
-
-        {appName && (
+      {scope !== 'page' && (
```

Comment:
Please review this part of the code as well:
https://github.com/vtex-apps/admin-pages/pull/300/files#diff-0bc6b796c2420f88ee9aff202ca8c4f7R124-R131

Reply:
You could replace `typeof scope` with either `Pick<typeof scope, 'sitewide' | 'template'>` or `Omit<typeof scope, 'page'>`.